src/deepsparse/server/cli.py

```diff
@@ -156,6 +156,15 @@ def config(config_path: str, host: str, port: int, log_level: str):
     default="info",
     help="Sets the logging level.",
 )
+@click.option(
+    "--loggers",
```

Comment:
To make it more explicit that its a file?
```suggestion
    "--loggers-path",
```

Reply:
want to keep the name consistent with the server config field since the rest of the args are